content/docs/user-guide/gems/reference/debug/imgui.md

```diff
@@ -1,8 +1,102 @@
----
-linkTitle: Immediate Mode GUI (IMGUI)
-title: Immediate Mode GUI (IMGUI) Gem
-description: The Immediate Mode GUI (IMGUI) Gem provides the 3rdParty library IMGUI which can be used to create runtime immediate mode overlays for debugging and profiling in Open 3D Engine (O3DE) projects.
-toc: true
----
-
-The Immediate Mode GUI (IMGUI) Gem provides the 3rdParty library IMGUI which can be used to create run time immediate mode overlays for debugging and profiling information in Open 3D Engine. Atom ImGUI is used by the PhysX debugging tools.
+---
+linkTitle: Immediate Mode GUI (ImGui)
+title: Immediate Mode GUI (ImGui) Gem
+description: The Immediate Mode GUI (ImGui) Gem provides the 3rdParty library IMGUI which can be used to create runtime immediate mode overlays for debugging and profiling in Open 3D Engine (O3DE) projects.
+toc: true
+---
+
+The Immediate Mode GUI (ImGui) Gem provides the 3rdParty library [Dear ImGui](https://github.com/ocornut/imgui), which can be used to create run time immediate mode menus and overlays for debugging and profiling information in Open 3D Engine. You can use it to access existing O3DE tooling or set up your own ImGui menus for your components or other game code.
+
+One of the great things about ImGui is that there is next to no overhead when the ImGui game overlay is not visible, which is the default state. It is also very lightweight while in use. 
+
+## Accessing ImGui menus
+
+ImGui can be accessed by pressing the `HOME` key in the editor, game or server launchers.
+
+![ImGui Menus in Viewport](/images/user-guide/gems/reference/debug/imgui_menus.png)
+
+> Server launchers support ImGui if rendering is active. ImGui requires the `rhi` cvar to not be null.
+
+You can use a keyboard, mouse or controller to navigate the ImGui menus.
+
+### Keyboard navigation
+For keyboard use the arrow keys to navigate the menus and `space` to activate or select an option.
+
+### Controller navigation
+Controller support is disabled by default, to turn it on you can use either the `imgui_EnableController` or `imgui_EnableControllerMouse` [CVARs](#cvars) to simulate the mouse with the controller.
+
+If using the controller then the following inputs are supported:
```

Comment:
Suggestion for clarity. The word "controller" alone might be a bit vague for some ESL readers because of its primary definition.

```suggestion
### Game controller navigation

You can use the `imgui_EnableController` [CVAR](#cvars) to enable game controller input support, or use the `imgui_EnableControllerMouse` [CVAR](#cvars) to simulate mouse input with a game controller.

The following inputs are supported when using a game controller:
```